src/plugin/memory-webview-main.ts

```diff
@@ -77,6 +94,10 @@ export class MemoryWebview implements vscode.CustomReadonlyEditorProvider {
     }
 
     public activate(context: vscode.ExtensionContext): void {
+        const getVisibleColumns =
+            (columnContext: string): string[] =>
+                columnContext.substring(1, columnContext.length - 1).split('|');
```

Comment:
See comment about surrounding pipes. This could perhaps also be moved to the common area and paired with a `toVisibleColumns` utility taking a `string[]` so that the two would form a more transparent protocol.